hail/Makefile

```
@@ -142,6 +142,8 @@ python/README.md: ../README.md
 $(PYTHON_JAR): shadowJar
```

Outside of scope here, but this rule should be:
```suggestion
$(PYTHON_JAR): $(SHADOW_JAR)
```